scripts/dataset_processing/tts/sfbilingual/ds_conf/ds_for_fastpitch_align.yaml

```diff
@@ -3,7 +3,7 @@ name: "ds_for_fastpitch_align"
 manifest_filepath: "train_manifest.json"
 sup_data_path: "sup_data"
 sup_data_types: [ "align_prior_matrix", "pitch" ]
-whitelist_path: "nemo_text_processing/text_normalization/zh/data/whitelist/default.tsv"
+whitelist_path: null # default whitelist will be used from https://raw.githubusercontent.com/NVIDIA/NeMo-text-processing/main/nemo_text_processing/text_normalization/zh/data/whitelist/default.tsv
```

Comment:
we have imported NeMo-Text-Processing, so the namespace should be in the NeMo namespace. Is there an elegant way to assign the file location in YAML?

The current implementation directly loads local files. Specifying an URL of the file requires adding a function that could parse and download the specified file.

Reply:
discussed offline. decided to remove this argument and instead, choose default `tts.tsv`.